### ISSUE TITLE
Introduce BCD, Scoring System

### ISSUE THREAD

**Issue by XoToM** [gbdev/gb-asm-tutorial#81]
While working on #42 I've realised that the Unbricked game for part 2 doesn't have any scoring at the moment, and it was briefly mentioned in #39 when a game over screen was brought up. I've also noticed that there isn't much information around on how the BCD instructions work, which gave me an idea;
What if a new lesson was added which would teach what BCD is and how to use BCD to display numbers? Displaying a score on the screen is probably the ideal use case for BCD, and it could also serve as a good resource on how to use BCD on the gameboy, especially since there isn't much information about it anywhere else. Tracking the score would be moved from the serial transfer lesson to here, and the serial transfer lesson could implement a simple leaderboard instead.

**Comment by avivace** [comment]
I agree this could be a generic "lesson" on BCD, rendering decimals and ultimately using those notions to implement a basic scoring system.

The Serial Transfer lesson can then give those for done and focus on the technicality of serial and exchanging data.

**Comment by ISSOtm** [comment]
Great idea! Thank you :)
